platformpolicy/keyprocessor.go

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package platformpolicy
+
+import (
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/platformpolicy/commoncrypto"
+	"github.com/insolar/insolar/platformpolicy/customcrypto"
+)
+
+func NewKeyProcessor() insolar.KeyProcessor {
```

Comment:
Why you copy pasted two implementations since you could just add parameter (curve) here?

Reply:
It is not copy past. They eat different crypto libs. Look at imports.

Reply:
Why we cant just use backported lib in both implementations?